apps/playground/src/main.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './main.css';
+import { styled } from 'linaria/react';
+import { useTheme } from '@itwin/itwinui-react';
+import App from './App';
+
+const Shell = () => {
+  const [theme, setTheme] = React.useState<'light' | 'dark'>('dark');
```

Comment:
Why not "os"?

Reply:
Why default dark?

Reply:
The `dark` was just personal preference when I made this playground. I've replaced it with a media query, which is easier to use than `os` for switching themes